### PR TITLE
Upgrade Android background upload service to 3.5

### PR DESCRIPTION
**Pull request by StevePotter**
DO NOT MERGE, not tested enough

### CODE REVIEW

**Comment by phsumi** [comment]
FYI, it's 3.5.1 now

File: README.md [hunk 1]
```diff
@@ -1,12 +1,10 @@
 # react-native-background-upload [![npm version](https://badge.fury.io/js/react-native-background-upload.svg)](https://badge.fury.io/js/react-native-background-upload)
 The only React Native http post file uploader with android and iOS background support.  If you are uploading large files like videos, use this so your users can background your app during a long upload.
 
-NOTE: Use major version 4 with RN 47.0 and greater.  If you have RN less than 47, use 3.0.  To view all available versions:
-`npm show react-native-background-upload versions`
-
-
 # Installation
 
+If you have React Native < 0.47, use version 3.0 of this library and good luck.
```
Comment (TSMMark) [inline README.md]: 🍀

**Comment by HeffZilla** [comment]
Any progress on this? Having build issues until this library is AndroidX compatible.

**Comment by HeffZilla** [comment]
I see you're looking for help testing this. How can I help?

**Comment by osrl** [comment]
It's 4.0 Beta now and it has breaking changes

**Comment by reime005** [comment]
closed because already upgraded

**Pull request closed without merging**